powerauth-java-server/src/main/java/io/getlime/security/powerauth/app/server/controller/api/IntegrationController.java

```diff
@@ -89,10 +97,13 @@ public ObjectResponse<GetIntegrationListResponse> getIntegrationList() throws Ex
      * @throws Exception In case the service throws exception.
      */
     @PostMapping("/remove")
-    public ObjectResponse<RemoveIntegrationResponse> removeIntegration(@RequestBody ObjectRequest<RemoveIntegrationRequest> request) throws Exception {
-        logger.info("RemoveIntegrationRequest received: {}", request);
-        final ObjectResponse<RemoveIntegrationResponse> response = new ObjectResponse<>(service.removeIntegration(request.getRequestObject()));
-        logger.info("RemoveIntegrationRequest succeeded: {}", response);
+    public ObjectResponse<RemoveIntegrationResponse> removeIntegration(@Valid @RequestBody ObjectRequest<RemoveIntegrationRequest> request) throws Exception {
+        final RemoveIntegrationRequest req = request.getRequestObject();
+        logger.info("action: removeIntegration, state: initiated, id: {}", req.getId());
```

Comment:
Maybe make the id parameter more descriptive
```suggestion
        logger.info("action: removeIntegration, state: initiated, integrationId: {}", req.getId());
```